torch/csrc/Event.cpp

```diff
@@ -49,21 +49,20 @@ static PyObject* THPEvent_pynew(
   }
 
   THPEvent* self = reinterpret_cast<THPEvent*>(ptr.get());
-  self->weakreflist = nullptr;
 
   // TODO: blocking and interprocess are not supported yet. To support them, the
   // flag system of c10::Event needs to be refactored. C10::Event should also
   // provide a generic constructor to support blocking and interprocess events.
   (void)blocking;
   (void)interprocess;
 
+  // LITERALINCLUDE START: PYTORCH EVENT CTOR
```

Comment:
Hey, we can't modify any code outside of torch_openreg for documentation purposes.